js/src/sdk/base.toolset.ts

```diff
@@ -60,12 +108,6 @@ export class ComposioToolSet {
 
     }
 
-    async getExpectedParamsForUser(
```

Comment:
Keep it here, if anything let's deprecate it and remove it sometime in the future to avoid breaking running code

Reply:
Done